origin.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>  
+        <header class="navbar">
+            <h3>Hector Andre Lau</h3>
+            <img src="./images/Icon-Menumenuicon.svg" alt="Menu">
+        </header>    
+        
+            <h1>Hey there, I'm Hector Andre Lau</h1>
+            <h2>I'm a software developer with experience in finance and logistic processes.</h2>
+
+        <p>I can build a website meeting the expected requirements, pluss I can add my actual knowledge of how the process need to work on certain areas like logistic, account, finance and billing and all of this focus to the good experience and satisfaction ot the customer.</p>
+        <div>
+                <h2>Let's get in touch...!</h2>
+                <div>
+                <img src="./images/facebooklogo.jpg" alt="facebooklogo">
+                <img src="./images/instagramlogo.jpg" alt="instagramlogo">
+                <img src="./images/linkedinlogo.jpg" alt="linkedinlogo">
+                <img src="./images/twitterlogo.png" alt="twitterlogo">
+                <img src="./images/githublogo.png" alt="githublogo">
```

Comment:
- [ ] Kindly use icons specified in the design, doing this will make your design look like the original design 🤗